### PR TITLE
[13.0][FIX] document_url: Fix module dependencies.

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
ping @CarlosRoca13 @chienandalu

### CODE REVIEW

File: document_url/__manifest__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/knowledge",
     "license": "AGPL-3",
+    "depends": ["web"],
```
Comment (pedrobaeza) [inline document_url/__manifest__.py]: Shouldn't it be mail?

Comment (sergio-teruel) [reply document_url/__manifest__.py]: I do not see that it is necessary... but I am not sure..

Comment (pedrobaeza) [reply document_url/__manifest__.py]: Seeing this:

https://github.com/OCA/knowledge/blob/f79f13724ff1aed24b83f23b2a79bf42ef22ae16/document_url/static/src/js/url.js#L11

I think it should be.

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done!! :smile:

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**